src/main/java/org/cru/godtools/domain/images/ImageService.java

```diff
@@ -20,18 +18,31 @@ public ImageService(Connection sqlConnection)
 
     public Image selectById(UUID id)
     {
-        return sqlConnection.createQuery(ImageQueries.selectById)
-                .setAutoDeriveColumnNames(true)
-                .addParameter("id", id)
-                .executeAndFetchFirst(Image.class);
+		Image image = sqlConnection.createQuery(ImageQueries.selectById)
+				.setAutoDeriveColumnNames(true)
+				.addParameter("id", id)
+				.executeAndFetchFirst(Image.class);
+
+		if(image != null && image.getImageContent() != null)
```

Comment:
Just a nit pick: what do you think about having a method that does this ``if``? It the same logic on line 43.